src/rendering/renderable.cpp

```diff
@@ -140,6 +163,12 @@ Renderable::Renderable(const ghoul::Dictionary& dictionary)
     // set type for UI
     _renderableType = p.type.value_or(_renderableType);
     addProperty(_renderableType);
+
+    //only used by a few classes such as RenderableTrail and RenderableSphere
+    if (p.renderBinMode.has_value()) {
+        setRenderBin(codegen::map<Renderable::RenderBin>(*p.renderBinMode));
+    }
+
```

Comment:
again, nitpicky, but I would remove this extra line here

(and add a space after the `//` in the comment on line 167)